verification/pmp-tests/utility.h

```diff
@@ -12,6 +12,12 @@
     #define EXT_ADDR_CLEAR  0xFFFFFFF8
     #define MAGIC_ADDR      0xFFFFFFFC
     
+    // G means (G + 2) bits from the 32 bit address are chopped off for matching
+    // i.e. G = 14 means 16 bits are removed
+    #define G 14
```

Comment:
It might be worth putting this info in the yml file, the nomenclature is a bit confusing as is.

Also I was able to successfully run the examples changing this so it seems to work well. For future reference, this command can be used to run all of the tests after building the core and the executables: `cp pmp_csr.bin meminit.bin && ../../rvb_out/sim-verilator/Vtop_core meminit.bin && cp pmp_na4.bin meminit.bin && ../../rvb_out/sim-verilator/Vtop_core meminit.bin && cp pmp_napot.bin meminit.bin && ../../rvb_out/sim-verilator/Vtop_core meminit.bin && cp pmp_tor.bin meminit.bin && ../../rvb_out/sim-verilator/Vtop_core meminit.bin`

Reply:
Agree. Would it be better to specify "G" directly, or should we give some human-readable byte granularity? e.g. a parameter like `pmp_minimum_granularity = "64K"` and have the python script calculate G?